ISISComputingGroup/ibex_gui
# Ticket 2421: Fixed bug in setting run control from GUI

### Description of work
This relates to https://github.com/ISISComputingGroup/IBEX/issues/2421

The underlying reason for the problem was that PVs were created, then immediately written to without having time to connect. The problem was highlighted by https://github.com/ISISComputingGroup/IBEX/issues/2298.

### To test
* Create a config with some run controlled blocks

In the run control dialog confirm that:
* You can change low limits / high limits / enabled and send them to the server
* You can revert all run control back to that of the configuration

---

#### Code Review

    - Manual system tests for the functionality should be added if there are no automated tests
    - Manual system tests can be removed from the template if they are covered by suitable automated tests

### Functional Tests


### Final Steps
